packages/hash/integration/src/mockData/index.ts

```diff
@@ -65,11 +64,6 @@ void (async () => {
     createOrgs(db),
   ]);
 
-  await createEntityTypes(db)([
```

Comment:
Were these types not used for any of the integration tests? It's nice we can get rid of it, but interesting we never used this

Reply:
No – we had an example `Task` type which we put in everyone's namespace. This meant that a list of 'all types' contained 'Task' multiple times, which was pretty silly, and it doesn't seem important to have this dummy data for testing (people can create their own type if they want one).